graphdatascience/tests/conftest.py

```diff
@@ -3,6 +3,9 @@
 
 def pytest_addoption(parser: Any) -> None:
     parser.addoption("--include-enterprise", action="store_true", help="include tests requring GDS enterprise")
+    parser.addoption(
+        "--encrypted", action="store_true", help="runs only tests requiring an encrypted connection to Neo4j"
```

Comment:
If we really want the "only" behavior I think `--only-encrypted` would be clearer

Reply:
`encrypted-only` :)